Khan/perseus
# [Locked Figure Labels] Add/edit/delete locked line labels

## Summary:
Add the ability to add/edit/delete locked line labels in the
interactive graph editor.

Note: locked labels within the points that define a locked line
are coming in a follow-up PR.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2347

## Test plan:
- Go to http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--mafs-with-locked-line-labels-flag
- Go to locked figures
- Open the locked line settings
- Confirm that the label settings are there
- Play around with the label settings and confirm that it moves
and updates with the line

<img width="371" alt="image" src="https://github.com/user-attachments/assets/e7fd912b-37f1-4179-a3ac-aa7e3c3fd2ff">
